File: tests/go/cases/case_204_container_handoff_global_01.go
package escape_tests

var retainedCase204 = []map[string]string{}

func Case204ContainerHandoffGlobal01(input string) string {
	raw := input
	if raw == "" {
		raw = "sample"
	}
	payload := map[string]string{
		"task": "container_handoff_global_01",
		"entity": "extreme",
		"stage": "stress",
		"input": raw,
		"checksum": "container_handoff_global_01:" + raw,
	}
	box := map[string]map[string]string{"v": payload}
	// ESCAPE: nested container handoff retained globally.
	retainedCase204 = append(retainedCase204, box["v"])
	return "ok"
}

File: tests/nodejs/cases/case_204_container_handoff_global_01.js
/**
 * Task case 204: container_handoff_global_01 deep stress pattern.
 */

const retainedCache = new Map();
const retainedAudit = [];
const retainedHandlers = [];

function case204ContainerHandoffGlobal01(input) {
  const taskName = 'container_handoff_global_01';
  const raw = input || 'sample';
  const payload = {
    task: taskName,
    entity: 'extreme',
    stage: 'stress',
    input: raw,
    checksum: `${taskName}:${raw.length}`,
  };
  const box = { deep: { value: payload } };
  // ESCAPE: nested indirection retained globally.
  retainedCache.set('nested', box.deep.value);
  return 'ok';
}

module.exports = { case204ContainerHandoffGlobal01 };
